src/scripts/main.js

```diff
@@ -1,3 +1,60 @@
 'use strict';
 
-// write your code here
+// import:
+const {
+  mobileSwipes,
+} = require('./mobile');
+
+const {
+  arrowsPlayHandler,
+  createItemInRandomEmptyField,
+} = require('./move');
+
+// vars:
+const gameScore = document.querySelector('.game-score');
+const gameField = document.querySelector('.game-field');
+const allCells = gameField.querySelectorAll('.field-cell');
+const startBtn = document.querySelector('.button');
+const highestScore = document.querySelector('.highest-score');
+
+// main function:
+function gameInit() {
+  highestScore.innerText = localStorage.getItem('score') || 0;
```

Comment:
It's okay since you saving only one item in localStorage, but usually we have separated method for that purpose that will return item from it (or null if there is nothing there)